nixos/modules/services/databases/postgresql.nix

```diff
@@ -106,6 +106,7 @@ in
       identMap = mkOption {
         type = types.lines;
         default = "";
+        example = "restya-board-users restya-board restya_board";
```

Comment:
Not sure if that'll help much, especially the last two elements. Also, the general form is described right below in the description.

Reply:
The description is open to interpretation while the example is somethink you know you can copy/paste. I think it's useful/complementary.

Reply:
It's a one-liner you can also copy-paste, no? Also, I think it's less open to interpretation because it exactly specifies what's needed at which position (even though escapes in SQL are needed, db names with dashes exist and now I have to guess what `restya-board` and `restya_board` are supposed to be or to find out - well - look at the description which is what you need to do already?).

I think I still stand with my previous point, I don't see any benefit, but rather the opposite.

Reply:
> It's a one-liner you can also copy-paste, no?

I am talking from the point of view of someone who doesn't know postgres, something I like to do and that is possible only thanks to nixos is enable a module and go as far as possible without the official doc, just with `man configuration.nix`. What I like about the "example" field is that I know I can copy/paste it and the format will be correct (while in description, it's mixed with other things so there is a doubt about what to copy paste). From that point even duplicating the example makes sense (though postgresql is one of the moduels for which that didn't work: it's so complex I had to read the official doc xD). Maybe a better example would be
```
map-name system-username database-username
map-name1 system-username1 database-username1
```
?
And I would instead put a link towards the doc in the description 
https://www.postgresql.org/docs/current/auth-username-maps.html . 
What do you think ?

Reply:
> What I like about the "example" field is that I know I can copy/paste it and the format will be correct (while in description, it's mixed with other things so there is a doubt about what to copy paste).

While I personally can't relate at all, I think you have a point: when "scanning" `configuration.nix(5)`, people probably train themselves to look at the examples and defaults primarily, so adding such a value would be fine for me.

It should be as generic as the description though, your current example is mostly confusing IMHO (especially if you have no idea what restya is - that's what I stumbled upon initially and system user / dbname cannot be distinguished at all there).

>  https://www.postgresql.org/docs/current/auth-username-maps.html .

Sounds reasonable :+1:

Reply:
done. Let me know if that's better